### PR TITLE
Добавлена кнопка Генерации улицы(Street)

### PR DESCRIPTION
**Pull request by MrConnorr**
default
Константинов Роман

### CODE REVIEW

**Comment by CrazyFlasher** [comment]
Please, resolve conflicts

File: examples/simpleMVC/dist/dependencies.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+dependencies=C:/Users/Connor/.m2/repository/com/domwires/core/0.9.9-SNAPSHOT/core-0.9.9-SNAPSHOT.swc,C:/Users/Connor/.m2/repository/com/domwires/starlingApp/0.9.12-SNAPSHOT/starlingApp-0.9.12-SNAPSHOT.swc,C:/Users/Connor/.m2/repository/com/gamua/starling/2.3.0/starling-2.3.0.swc,C:/Users/Connor/.m2/repository/com/feathersui/feathers/3.1.1/feathers-3.1.1.swc,C:/Users/Connor/.m2/repository/com/feathersui/MetalWorksMobileTheme/3.1.1/MetalWorksMobileTheme-3.1.1.swc
```
Comment (CrazyFlasher) [inline examples/simpleMVC/dist/dependencies.properties]: remove this file from repository

File: examples/simpleMVC/dist/pom.xml.dependencies [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (CrazyFlasher) [inline examples/simpleMVC/dist/pom.xml.dependencies]: remove this file from repository

File: examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as [hunk 3]
```diff
@@ -42,12 +43,14 @@ package com.domwires.example.simplemvc.view
 			firstNameLabel = createForm("First name:", layoutGroup, onFirstNameClicked);
 			lastNameLabel = createForm("Last name:", layoutGroup, onLastNameClicked);
 			ageLabel = createForm("Age:", layoutGroup, onAgeClicked);
+			streetLabel=createForm("Street",layoutGroup,onStreetClicked);
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as]: Use same code format (don forget spaces)

File: examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as [hunk 4]
```diff
@@ -65,6 +68,11 @@ package com.domwires.example.simplemvc.view
 			firstNameLabel.text = model.firstName;
 		}
 
+		private function streetChanged(m:IMessage):void
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as]: add speces

File: examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppViewMessage.as [hunk 5]
```diff
@@ -10,6 +10,7 @@ package com.domwires.example.simplemvc.view
 		public static const FIRST_NAME_CLICKED:AppViewMessage = new AppViewMessage("FIRST_NAME_CLICKED");
 		public static const LAST_NAME_CLICKED:AppViewMessage = new AppViewMessage("LAST_NAME_CLICKED");
 		public static const AGE_CLICKED:AppViewMessage = new AppViewMessage("AGE_CLICKED");
+		public static const STREET_CLICKED:AppViewMessage=new AppViewMessage("STREET_CLICKED");
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppViewMessage.as]: spaces again

**Comment by MrConnorr** [comment]
Resolved conflicts

**Pull request closed without merging**